orchagent/muxorch.cpp

```diff
@@ -1631,14 +1637,28 @@ void MuxOrch::update(SubjectType type, void *cntx)
             }
             else
             {
-                updateNeighbor(*update);
+                try
+                {
+                    updateNeighbor(*update);
+                }
+                catch (const std::exception& e)
+                {
+                    SWSS_LOG_ERROR("Exception caught while updating neighbor");
+                }
             }
             break;
         }
         case SUBJECT_TYPE_FDB_CHANGE:
         {
             FdbUpdate *update = static_cast<FdbUpdate *>(cntx);
-            updateFdb(*update);
+            try
+            {
+                updateFdb(*update);
+            }
+            catch (const std::exception& e)
+            {
+                SWSS_LOG_ERROR("Exception caught while updating FDB");
```

Comment:
It is dangerous to swallow general exception?

Reply:
this is a specific case which is not handled in general exception handling flow. also, in this scenario, we dont want orchagent to exit for some unknown exception.